Assets/Mirror/Examples/Basic/Scripts/Player.cs

```diff
@@ -1,77 +1,129 @@
+using System.Collections.Generic;
 using UnityEngine;
-using UnityEngine.UI;
 
 namespace Mirror.Examples.Basic
 {
     public class Player : NetworkBehaviour
     {
-        [Header("Player Components")]
-        public RectTransform rectTransform;
-        public Image image;
+        // Players List used by BasicNetManager to manage playerNumber
+        internal readonly static List<Player> playersList = new List<Player>();
```

Comment:
Could this list instead be in BasicNetManager so we dont need the static field?

Reply:
Done